ATFConverter/cdli_import.py

```diff
@@ -49,12 +51,155 @@ def __read_file__(file):
             return line_output
 
     @staticmethod
-    def discern_tablets(__read_file__):
+    def __discern_texts__(read_file):
         """
-        Using the __read_file__ function, this process separates out tablets
+        Using the read_file function, this process recognizes tablets
         based off of metadata in the file.
-        :param __read_file__: List of strings made up of individual lines of
-        file downloaded from CDLI.
-        :return: list that separates out disparate texts into lists of strings
-        made up of individual lines of file downloaded from CDLI.
+        :param read_file: This is the text file that you downloaded from
+        CDLI.
+        :return: List that notes disparate texts in the downloaded file.
+        """
+        output = [line for line in read_file if line.startswith('&')]
+        return output
+
+    @staticmethod
+    def __split_texts__(read_file):
+        """
+        Using read_file, this process separates out tablets based off of
+        metadata in the file.
+        :param read_file: This is the text file that you downloaded from
+        CDLI.
+        :return: List that separates out disparate texts into lists of strings.
+        """
+        output = []
+        for line in read_file:
+            if line.startswith('Primary'):
+                yield output
+                output = []
+            output.append(line)
+        yield output    # issue: prints "[[]" before list
```

Comment:
Here's a compact way to split the file into a list of lists:
```
texts, text = [], []
for line in file_contents:
    if line.strip() == '':
        if len(text) > 0:
            texts.append(text)
        text = []
    else:
        text.append(line.strip())
```

Reply:
In replacing this, i'm getting "None." in return -- I'll be working on this problem tonight